auslib/db.py

```diff
@@ -1847,17 +1844,25 @@ def getReleaseInfo(self, names=None, product=None, limit=None, transaction=None,
                 ((self.db.releases.name == self.db.rules.mapping) | (self.db.releases.name == self.db.rules.fallbackMapping)),
             )
             if transaction:
-                ref_list = transaction.execute(select([self.db.releases.name, self.db.rules.rule_id]).select_from(j)).fetchall()
+                ref_list = transaction.execute(
```

Comment:
Nit: I remember the current pattern is used in the code base, but I just realized we can simplify it with:

```py
transaction = transaction if transaction else self.getEngine()
ref_list = transaction.execute(
    select([
        self.db.releases.name, self.db.rules.rule_id, self.db.rules.product, self.db.rules.channel
    ]).select_from(j)
).fetchall()
```

What do you think?

Reply:
SGTM!